README.md

```diff
@@ -5,6 +5,13 @@ Some series in Cinemeta are ordered based on original air date
 and does not match the order that is used in the torrent scene. 
 This happens for a lot of Netflix shows, when they are bought and split up.
 
+## TMDB Series
+
+Netflix series that are added to avoid Episodes Conflict.
```

Comment:
I don't think it should state Netflix series. Just series should be enough, as not all of them are/might be Netflix's.